Library/Formula/zeromq.rb

```diff
@@ -48,6 +50,10 @@ def install
       args << "--without-libsodium"
     end
 
+    if build.with? "norm"
```

Comment:
Since there's no `without` option being passed to `configure` here this can be single-line:

```ruby
args << "--with-norm" if build.with? "norm"
```